content/articles/google-identity-provider.md

```diff
@@ -86,7 +86,7 @@ When you delete or suspend a team member from your company's Google Workspace ad
 
```

Comment:
It looks like this should be moved here:

```
![screenshot: remove Google identity success](/files/remove-google-identity-success.png)
```

Reply:
Made the change here: https://github.com/dnsimple/dnsimple-support/pull/1248/commits/d6f19ca59d9878f2efb40b06e9a203e91ad7db86